### DISCUSSION TITLE
Empty string „Default“ for a VARCHAR Column shown as '' ?

### DISCUSSION THREAD

**Discussion by oliveratgithub** [Sequel-Ace/Sequel-Ace#2127] [Q&A] [answered]
Sorry for posting this but I am a bit confused by it right now (checked the Sequel Ace Settings and couldn't find a relevant option).

Is it default behaviour that an empty default value - but not NULL - for a MySQL column of type VARCHAR is visually indicated by the App's UI as:

`''`

? I am literally trying to figure out if that's an actual issue in my DB, or just a "UI Feature"…

If the latter: I would personally prefer just seeing a blank.

**Comment by Jason-Morcos** [comment, marked as answer]
UI feature - default values that are strings are indicated as '' to make it clear that a default is set
BLANK means no default is set (not null, necessarily, since some values are not nullable)
NULL means null

**Reply by oliveratgithub** [reply]
Thanks. I was so confused by it, because when clicking into the Default value cell on the relevant rows, it will actually show `''` as highlighted text selection.

While `NULL` can be customized in the App Settings, would then most likely make sense to also have an option to adjust the behaviour for displaying empty strings. Will raise this as a feature request I guess.
